storage/innobase/log/log0crypt.cc

```diff
@@ -429,7 +429,7 @@ ATTRIBUTE_COLD bool log_crypt_read_checkpoint_buf(const byte* buf)
 
 /** Encrypt or decrypt a temporary file block.
 @param[in]	src		block to encrypt or decrypt
-@param[in]	size		size of the block
+@param[in]	size	length of both 'src' and `dst` blocks in bytes
```

Comment:
Do not use ' or ` in comments that refer to parameters, and do not break the indentation.

Reply:
This should be resolved